data_manager.rb

```diff
@@ -0,0 +1,109 @@
+require 'json'
+
+module DataManager
+  DATA_DIR = 'data'.freeze
+
+  def save_data(filename, data, &serialization_block)
+    return unless data
+
+    filepath = File.join(DATA_DIR, filename)
+    serialized_data = data.map { |item| serialization_block.call(item) }
+    File.open(filepath, 'w') do |file|
+      JSON.dump(serialized_data, file)
+    end
+  end
+
```

Comment:
- [ ] It would be nice if you follow the project requirements because it is clearly mentioned in the project requirements that your app should store the data into the JSON files when your app will be quitted 🤔 But right now it is not behaving in the similar way even it is saving the data into JSON files before quitting the console app 😨 Kindly fix this behavior of your app to follow the project requirements 👍